maddymz/irondb
# US172

Adds scss to override bootstrap to match: https://brandguide.asu.edu/Elements-of-the-brand/Color-palette
style.css changed to reflect non-bootstrap ASU brand styles such as typography.

To test:
1. Basic setup: After cloning, navigating to repo, and checking out to branch `US172` run: `npm install`, `gulp sass`, `gulp js`, `mkdir pg-data`
2. Check that there is a directory name `scss` is in the main folder.
3. Go to `localhost:3001/example` to view div bg-color test:

### **Success:**
![screen shot 2018-11-29 at 2 29 47 pm](https://user-images.githubusercontent.com/6512755/49247204-b20ba200-f3e4-11e8-93ab-3cc4b065bfca.jpg)

### **Failure:**
![screen shot 2018-11-29 at 2 29 15 pm](https://user-images.githubusercontent.com/6512755/49247268-d798ab80-f3e4-11e8-8737-cd6e450244fd.jpg)
